go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	cloud.google.com/go v0.78.0 // indirect
-	github.com/GoogleCloudPlatform/spark-on-k8s-operator v0.1.3
+	github.com/GoogleCloudPlatform/spark-on-k8s-operator v0.0.0-20200723154620-6f35a1152625
```

Comment:
thats a weird version, i guess we have no choice right now?

Reply:
Yeah, this is the latest sha on https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/tree/multi-version-support .